### DISCUSSION TITLE
renormalised 2nd order force constants

### DISCUSSION THREAD

**Discussion by dyang2103** [ttadano/alamode#183] [Q&A] [answered]
Dear developors and users,

I am using QE to calculate the IFCs. I am using almode to get the  renormalised  IFCs. Next, I want to use these renormalised force constants from almode in shengbte for thermal conductivity calculations. I have used scph to get the PREFIX.scph_dfc2. How can I convert it back to shengbte format?

Thank you very much.
Xun

**Comment by triphysics** [comment, marked as answer]
Take a look at
https://github.com/ttadano/alamode/blob/develop/tools/scph_to_qefc.py
Using this you can transform the renormalized IFCs back in the QE format. To use this in Shengbte set espresso= .true. in your CONTROL file.
